_posts/en/pages/2017-01-01-download.md

```diff
@@ -103,7 +103,7 @@ choosing_builders: >
 
 release_key_obtained: "The output of the command above should say that one key was imported, updated, has new signatures, or remained unchanged."
 
-verify_checksums_file: "Verify that the checksums file is PGP signed by the release signing key:"
+verify_checksums_file: "Verify that the checksums file is PGP signed by a trusted release signing key:"
```

Comment:
This implies that there is such a thing as a trusted key in the absolute sense. Suggested rephrase:

```suggestion
verify_checksums_file: "Verify that the checksums file is PGP signed by a sufficient amount of keys you trust and have imported into your keychain."
```

Reply:
ACK